Gemfile

```diff
@@ -25,7 +25,7 @@ gem 'sdoc', '~> 0.4.0', group: :doc
 
 gem 'dotenv-rails', '>= 2.0.1'
 
-gem 'activeadmin', github: 'activeadmin'
+gem 'activeadmin', '1.0.0.pre1'
```

Comment:
Yeah, definitely a good idea to keep everything as up-to-date as possible, so if we can use `pre2` and `pre2` is stable, we ought to use it.